news/2021-11-08-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,132 @@
+
+We intend to stay alert on these developments in the future and promptly address issues that arise in future updates. For this reason, we implore mappers and players to try leaning into these changes and have fun with them!
+
+### Rhythm complexity
+
+For over 7 years, star rating has been calculated based purely on the density (or speed) of the notes within any given map. 
+
+With [a new change](https://github.com/ppy/osu/pull/14395) submitted by [**Xexxar**](https://osu.ppy.sh/users/2773526), rhythm complexity will now play an important role in determining the star rating of a map. 
+
+This change introduces an algorithmic assessment of the complexity of note patterns in historical context and will introduce a more accurate assessment for maps that feature atypical combinations of triples, doubles, quints and much more!
+
+Consider these two rhythms:
+
+![](/wiki/shared/news/2021-11-08-performance-points-star-rating-updates/rhythm-complexity.png)
+
+Before this difficulty calculation change, the first pattern would have always had greater strain than the second.  In reality, the second can be as demanding or even harder (or potentially harder) than the first depending on the map. With these new changes, the algorithm now more accurately reflects this.
+
+These changes are exceedingly complex from a mathematical point of view. To keep things as simple as possible, notes are grouped up into 'islands', with a triple expressed as a size 3 'island'. Each island is then grouped by the algorithm and considered in a variety of ways, such as:
+
+- islands becoming smaller or larger (size 5 to size 3, for example)
+- islands which change from even to odd sizes
+- the ratio of time elapsed between islands (changes like 1/4 streams into 1/3 streams)
+
+These changes all collectively combine to produce a rhythm value which then adjusts the difficulty of a pattern, from which star rating is ultimately derived.
+
+We hope this change will bring a renewed interest in rhythmically complex songs on osu! and will incentivize players to improve their finger control. 
+
+Good luck setting new, impressive top plays that showcase your skills!
+
+### Total star rating adjustments
+
+In a nutshell, osu! difficulty calculation processes a map and produces skill rating values. These skill values are then manipulated in two different ways to produce two different numbers that you’ll be familiar with: star rating and performance points. 
+
+The correlation between these two numbers has now been improved, following the mathematical derivation work from [**Naitsirk**](https://osu.ppy.sh/users/8202998) in [this pull request](https://github.com/ppy/osu/pull/13986). 
+
+To summarize these changes briefly, maps which focus on one skill will now receive lower star rating than ones which focus on both. **These changes do not affect performance points.**
```

Comment:
While true, this feels like it's beside the point of the change. The main point I'd emphasise here is that the old SR didn't really care about what the *composition* of a star rating was - as in how large the individual skill values are; it just summed the constituent parts and left it at that, while the new one combines them in a way analogous to how pp does. The statement that one-skill maps will be nerfed is a corollary of the above.